### PR TITLE
tweak grammar and punctuation for clarity

### PR DESCRIPTION
**Pull request by TallTed**
### Why:

Observed incorrect grammar and punctuation.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

A few words and punctuation marks.

### Check off the following:

- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@TallTed Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by sabrowning1 — CHANGES_REQUESTED** [review]
@TallTed thanks so much for finding these grammatical errors and opening a PR! ✨  I've made a few changes below; please let me know when you've committed them and I'll get this merged.

**Review by sabrowning1 — APPROVED** [review]
👋🏼 @TallTed, thanks again for your help with these grammatical errors! I've applied my suggestions and will go ahead and merge this to get these errors fixed 🙂 if you have any questions on my changes, feel free to take a look at the notes I left in my comments.

Thank you for your help maintaining our docs! ✨

File: contributing/translations-for-writers.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ Use the following checklist to help make your files more translation-friendly. F
...
-| Avoid vague nouns and pronouns. | "Maintainers and contributors have access to files and comments. In the pull request they make changes to it." In this example it is not clear if the changes are being made to the file or to the comments. Another example “After saving the file in the folder, the user deleted it.” In this sentence it is not clear what was deleted (file or folder). | If a pronoun seems to refer to more than one antecedent, either reword the sentence to make the antecedent clear or replace the pronoun by a noun to eliminate ambiguity. |
+| Avoid vague nouns and pronouns. | "Maintainers and contributors have access to files and comments. In the pull request they make changes to it." In this example it is not clear if the changes are being made to the file or to the comments. Another example “After saving the file in the folder, the user deleted it.” In this sentence it is not clear what was deleted (file or folder). | If a pronoun seems to refer to more than one antecedent, either reword the sentence to make the antecedent clear or replace the pronoun with a noun to eliminate ambiguity. |
```
[the remark this answers is not in the document]
Comment (TallTed) [reply contributing/translations-for-writers.md]: Upon re-reading, I'd change `In this sentence, it is not clear what was deleted (a file or a folder).` to `In this sentence, it is not clear whether the user deleted the file or the folder.` Do you want another PR, or will you make this change directly?

File: contributing/translations-for-writers.md [hunk 2]
```diff
@@ -26,10 +26,10 @@ Use the following checklist to help make your files more translation-friendly. F
...
-| Avoid invisible plurals. | "Program update" or "File retrieval". Is this an update of one program or a general procedure for multiple programs?  For "File retrieval", Are you retrieving one file or all of them? | Write the sentence more clearly or provide additional context to eliminate ambiguity that can result in an incorrect translation. |
+| Avoid excessive use of stacked modifiers (noun strings). This can lead to incorrect translations, because it is not easy to tell what modifies what. | “public repository default source settings” or “OAuth app access restrictions” | "Default source settings for the public repository"  or "restrictions for OAuth app access." If using a stacked modifier is essential, make sure the background information and context are clear so the linguist can understand which noun is being modified. |
```
[the remark this answers is not in the document]
Comment (TallTed) [reply contributing/translations-for-writers.md]: I won't argue your changes. However, on rereading, I do suggest changing `"restrictions for OAuth app access."` to `"restrictions for app access via OAuth."` As earlier, do you want another PR, or would you change this directly?

**Comment by TallTed** [comment]
Your comments and merge came through while I was on holiday. Let me know whether you want a new PR for my comments above, or you prefer to apply them directly.

**Pull request merged**